packages/plugin-autocapture-browser/test/helpers.test.ts

```diff
@@ -83,11 +84,26 @@ describe('autocapture-plugin helpers', () => {
       expect(result).toEqual(false);
     });
 
-    test('should return false when element is a sensitive tag', () => {
+    test('should return true when element is a sensitive tag', () => {
       const element = document.createElement('a');
       const result = isNonSensitiveElement(element);
       expect(result).toEqual(true);
     });
+
+    test('should detect contenteditable as a sensitive tag', () => {
+      // Create the SVG element
+      const svg = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+      svg.setAttribute('width', '200');
+      svg.setAttribute('height', '200');
+      svg.setAttribute('viewBox', '0 0 200 200');
+
+      expect(isNonSensitiveElement(svg)).toEqual(true);
+      const element = document.createElement('div');
+      expect(isNonSensitiveElement(element)).toEqual(true);
+
+      element.setAttribute('contenteditable', 'true');
+      expect(isNonSensitiveElement(element)).toEqual(true);
```

Comment:
That was a typo, it is supposed to be false, the reason the test was not returning the correct result was because `.isContentEditable` is not supported by jsdom even though it would work in a browser. 

Either way, I now switched it to use a more universally supported approach:

```javascript
element.getAttribute('contenteditable')?.toLowerCase() === 'true'
```